pandas/tests/frame/methods/test_select_dtypes.py

```diff
@@ -82,6 +82,12 @@ def test_select_dtypes_exclude_include_using_list_like(self):
         e = df[["b", "c", "e"]]
         tm.assert_frame_equal(r, e)
 
+        exclude = (np.datetime64,)
+        include = np.bool_, "int"
+        r = df.select_dtypes(include=include, exclude=exclude)
+        e = df[["b", "e"]]
+        tm.assert_frame_equal(r, e)
```

Comment:
pls separate this out into a dedicated test with a descriptive name

Reply:
Ok